GetOwnershipClaimStatus/handler.ts

```diff
@@ -0,0 +1,48 @@
+import {
+  withRequestMiddlewares,
+  wrapRequestHandler
+} from "@pagopa/ts-commons/lib/request_middleware";
+import * as express from "express";
+import {
+  IResponseErrorInternal,
+  IResponseErrorNotFound,
+  IResponseSuccessJson,
+  ResponseErrorInternal
+} from "@pagopa/ts-commons/lib/responses";
+import { Context } from "@azure/functions";
+import {
+  NonEmptyString,
+  OrganizationFiscalCode
+} from "@pagopa/ts-commons/lib/strings";
+import { ContextMiddleware } from "@pagopa/io-functions-commons/dist/src/utils/middlewares/context_middleware";
+import { RequiredParamMiddleware } from "@pagopa/io-functions-commons/dist/src/utils/middlewares/required_param";
+import { ClaimProcedureStatus } from "../generated/definitions/ClaimProcedureStatus";
+
+type Handler = (
```

Comment:
Naming: `type Handler` and `const Handler` refer to two very different things, it's misleading fo a reader

Reply:
Maybe it's better a name like `ClaimProcedureStatusHandler`? Fix with 0ed84cc